app/web/components/Select.tsx

```diff
@@ -44,7 +50,7 @@ export default function Select<T extends Record<string | number, string>>({
     >
       <InputLabel htmlFor={id}>{label}</InputLabel>
       <MuiSelect
-        native
+        native={native}
```

Comment:
user can now choose

Reply:
Is this for the phone? What's "native"?

Reply:
native it's an option for Material components, to make it look like native components in the phone (like when openning a dropdown for example) still didn't confirm it, that's something I wanted to do in the testing of this PR